lib/subscriber.js

```diff
@@ -61,7 +61,10 @@ class SubscriptionContext {
   }
 
   close () {
-    this.queue.close()
+    /* istanbul ignore else */
+    if (typeof this.queue.close === 'function') {
+      this.queue.close()
```

Comment:
can you add a comment explaining this?

Reply:
Yes. BTW I've got another idea, we can set `close` to noop function in constructor to avoid that `if`.

Reply:
@mcollina added a comment.